File: siserp-logistica-backend/src/main/java/com/besoft/siserp/lgtc/dto/response/InventarioBuscarCompraResponse.java
package com.besoft.siserp.lgtc.dto.response;

import java.io.Serializable;
import java.util.List;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class InventarioBuscarCompraResponse implements Serializable {
	private static final long serialVersionUID = 1L;

	private String codCompra;
	private String nomProveedor;
	private Long tipDocProveedor;
	private String nroDocProveedor;
	private Double montoTotal;
	private String observacion;
	private List<InventarioBuscarCompraDetResponse> listaDetalleCompra;
}

File: siserp-logistica-frontend/src/main/siserp-logistica-frontend/src/app/modules/intranet/dto/response/inventario-buscar-compra.response.ts
import { InventarioBuscarCompraDetResponse } from "./inventario-buscar-compra-det.response";

export class InventarioBuscarCompraResponse {
    codCompra: string;
    nomProveedor: string;
    tipDocProveedor: number;
    nroDocProveedor: string;
    montoTotal: number;
    observacion: string;
    listaDetalleCompra: InventarioBuscarCompraDetResponse[];
}